src/classes/trip.js

```diff
@@ -0,0 +1,10 @@
+import time from '../scripts/time';
+
+class Trip {
+  constructor(tripInfo) {
+    this.tripLocation = tripInfo.destinyID;
+    this.tripStatus = time.isBetween(tripInfo.status, tripInfo.date.split('/'), time.daysFromDate(tripInfo.date.split('/'), tripInfo.duration))
```

Comment:
Once again, the king of obnoxiously long lines. 

Going to probably handle the split and refine this in scripts, just have this located here temporarily